recipe/meta.yaml

```diff
@@ -22,13 +22,14 @@ outputs:
       run:
         - python >=3.8
         - cartopy >=0.20
-        - cf-units >=3
-        - cftime >=1.5.0 
+        - cf-units >=3.1
+        - cftime >=1.5
         - dask-core >=2
         - matplotlib-base
-        - netcdf4 >=1.4
+        - netcdf4
         - numpy >=1.19
         - scipy
+        - shapely <1.8.3
         - python-xxhash
```

Comment:
Need `pyproj` in here too, based on the [Iris GitHub dependencies](https://github.com/SciTools/iris/tree/aa2dca38d3d462a5ca9133aa313c3f5285564bdf/requirements/ci).

Reply:
cda3b1fcc85f64b43d848b473a9a026d4941449a